Description: fix syntax in .Bd macros and some minor spelling errors
Author: Fabio Augusto De Muzio Tobich <ftobich@gmail.com>
Forwarded: no
Last-Update: 2020-12-19
Index: openresolv/resolvconf.conf.5.in
===================================================================
--- openresolv.orig/resolvconf.conf.5.in
+++ openresolv/resolvconf.conf.5.in
@@ -67,14 +67,14 @@ If set, only these interfaces will be pr
 If set, these interfaces will not be processed.
 .It Sy interface_order
 These interfaces will always be processed first.
-If unset, defaults to the following:-
-.Bd -compact -literal -offset indent
+If unset, defaults to the following:
+.Bd -literal -offset indent
 lo lo[0-9]*
 .Ed
 .It Sy dynamic_order
 These interfaces will be processed next, unless they have a metric.
-If unset, defaults to the following:-
-.Bd -compact -literal -offset indent
+If unset, defaults to the following:
+.Bd -literal -offset indent
 tap[0-9]* tun[0-9]* vpn vpn[0-9]* wg[0-9]* ppp[0-9]* ippp[0-9]*
 .Ed
 .It Sy inclusive_interfaces
@@ -83,8 +83,8 @@ This is handy when 3rd party integration
 .Nm resolvconf -x
 option and you want to disable it easily.
 .It Sy local_nameservers
-If unset, defaults to the following:-
-.Bd -compact -literal -offset indent
+If unset, defaults to the following:
+.Bd -literal -offset indent
 127.* 0.0.0.0 255.255.255.255 ::1
 .Ed
 .It Sy search_domains
@@ -128,24 +128,27 @@ The syntax is this:
 .Va $keyword Ns / Ns Va $match Ns / Ns Va $replacement
 .Pp
 Example, given this resolv.conf:
-.Bd -compact -literal -offset indent
+.Bd -literal -offset indent
 domain foo.org
 search foo.org dead.beef
 nameserver 1.2.3.4
 nameserver 2.3.4.5
 .Ed
-and this configuaration:
-.Bd -compact -literal -offset indent
+.Pp
+and this configuration:
+.Bd -literal -offset indent
 replace="search/foo*/bar.com"
 replace="$replace nameserver/1.2.3.4/5.6.7.8"
 replace="$replace nameserver/2.3.4.5/"
 .Ed
+.Pp
 you would get this resolv.conf instead:
-.Bd -compact -literal -offset indent
+.Bd -literal -offset indent
 domain foo.org
 search bar.com
 nameserver 5.6.7.8
 .Ed
+.Pp
 .It Sy replace_sub
 Works the same way as
 .Sy replace
@@ -156,11 +159,12 @@ Using the same example resolv.conf and c
 to
 .Sy replace_sub ,
 you would get this resolv.conf instead:
-.Bd -compact -literal -offset indent
+.Bd -literal -offset indent
 domain foo.org
 search bar.com dead.beef
 nameserver 5.6.7.8
 .Ed
+.Pp
 .It Sy state_dir
 Override the default state directory of
 .Pa @VARDIR@ .
@@ -224,7 +228,7 @@ in the subscribers main configuration fi
 .Pp
 To disable a subscriber, simply set it's name to NO.
 For example, to disable the libc subscriber you would set:
-.Bd -compact -literal -offset indent
+.Bd -literal -offset indent
 libc=NO
 .Ed
 .Bl -tag -width indent
@@ -234,14 +238,14 @@ This file tells dnsmasq which name serve
 This file tells dnsmasq which name servers to use for global lookups.
 .Pp
 Example resolvconf.conf for dnsmasq:
-.Bd -compact -literal -offset indent
+.Bd -literal -offset indent
 name_servers=127.0.0.1
 dnsmasq_conf=/etc/dnsmasq-conf.conf
 dnsmasq_resolv=/etc/dnsmasq-resolv.conf
 .Ed
 .Pp
 Example dnsmasq.conf:
-.Bd -compact -literal -offset indent
+.Bd -literal -offset indent
 listen-address=127.0.0.1
 # If dnsmasq is compiled for DBus then we can take
 # advantage of not having to restart dnsmasq.
@@ -257,14 +261,14 @@ Include this file in the named global sc
 This file tells named which name servers to use for specific domains.
 .Pp
 Example resolvconf.conf for named:
-.Bd -compact -literal -offset indent
+.Bd -literal -offset indent
 name_servers=127.0.0.1
 named_options=/etc/named-options.conf
 named_zones=/etc/named-zones.conf
 .Ed
 .Pp
 Example named.conf:
-.Bd -compact -literal -offset indent
+.Bd -literal -offset indent
 options {
 	listen-on { 127.0.0.1; };
 	include "/etc/named-options.conf";
@@ -285,14 +289,14 @@ If this variable is not set then it's wr
 .Pa pdnsd_conf .
 .Pp
 Example resolvconf.conf for pdnsd:
-.Bd -compact -literal -offset indent
+.Bd -literal -offset indent
 name_servers=127.0.0.1
 pdnsd_conf=/etc/pdnsd.conf
 # pdnsd_resolv=/etc/pdnsd-resolv.conf
 .Ed
 .Pp
 Example pdnsd.conf:
-.Bd -compact -literal -offset indent
+.Bd -literal -offset indent
 global {
 	server_ip = 127.0.0.1;
 	status_ctl = on;
@@ -308,13 +312,13 @@ server {
 This file tells pdns_recursor about specific and global name servers.
 .Pp
 Example resolvconf.conf for pdns_recursor:
-.Bd -compact -literal -offset indent
+.Bd -literal -offset indent
 name_servers=127.0.0.1
 pdns_zones=/etc/pdns/recursor-zones.conf
 .Ed
 .Pp
 Example recursor.conf:
-.Bd -compact -literal -offset indent
+.Bd -literal -offset indent
 allow-from=127.0.0.0/8, ::1/128
 forward-zones-file=/etc/pdns/recursor-zones.conf
 .Ed
@@ -324,13 +328,13 @@ This file tells unbound about specific a
 When set to YES, unbound marks the domains as insecure, thus ignoring DNSSEC.
 .Pp
 Example resolvconf.conf for unbound:
-.Bd -compact -literal -offset indent
+.Bd -literal -offset indent
 name_servers=127.0.0.1
 unbound_conf=/etc/unbound-resolvconf.conf
 .Ed
 .Pp
 Example unbound.conf:
-.Bd -compact -literal -offset indent
+.Bd -literal -offset indent
 include: /etc/unbound-resolvconf.conf
 .Ed
 .El
